### ISSUE TITLE
Trying to spin during the emote cooldown after doing another emote makes you dizzy

### ISSUE THREAD

**Issue by GoldenAlpharex** [Skyrat-SS13/Skyrat-tg#2685]
Reporting client version: 513.1539

## Round ID:
489

## Testmerges:
- [Automatically flag Severe+ intensity ambitions for admin review](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2421)
- [[TM FIRST] Wounds are applied easier on damaged bodyparts](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2406)
- [[Fully Modular] Xenomorph Balancing - Buffs & Speeding Tickets](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2180)
- [Makes command immune to FAMILIES](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2175)
- [[TM] Disables sentient disease - pending rebalancing/tweaks](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2030)
- [Brig timers now calculate from real time, not time dilation time. You can set brig timers to 60 minutes](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1939)
- [[DNM] Cryo on core maps [TESTMERGE ONLY]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1783)
- [[WIP] Combat rework - less lethality, less hardstuns, nonlethal takedowns. Focus on combat ineffectivity](https://github.com/Skyrat-SS13/Skyrat-tg/pull/1285)
- [[TESTMERGE] Makes icon2html not work](https://github.com/Skyrat-SS13/Skyrat-tg/pull/948)
- [Revert "Slimeperson race improvements - digitigrade legs, transparent markings/mutant bodyparts"](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2673)

## Reproduction: 
Basically, when you try to spin right after doing another emote (doesn't really seem to matter which), it makes you dizzy, even though there's no reason to, because you're not even spinning in the first place, yet it states in chat that "You feel woozy from spinning". It might be because it's not tied to the actual emotes being used, but some sort of cooldown that might be shared with what Skyrat has implemented for it.

I'd honestly suggest removing the woozyness caused from spinning, as that was a /tg/ thing to try and deal with people spamming the spin, just like with how spamming the flips makes you stumble and take some damage, which wouldn't really be an issue on Skyrat, where there's a cooldown for _all_ the emotes already.

**Comment by atakiya** [comment]
I wouldn't remove it, as we may lift cooldowns at a later point (no promises though).
Best would be to check for the cooldown before running the woozyness, which can be done.

Thanks for the report.

**Comment by Zergspower** [comment]
seems the time between is only a couple of seconds but is now also purely RNG beyond that - potentially Feature now?
